kafka-client-lib/pom.xml

```diff
@@ -17,7 +17,7 @@
         <dependency>
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka_2.12</artifactId>
-            <version>1.1.0</version>
+            <version>2.2.1</version>
```

Comment:
I forget, is 2.3 still not available?

Reply:
@afalko 2.3 has slf4j dependency conflict with Jenkins core. Atm I'm creating PR to bump slf4j version on Jenkins core